src/plugins/auto/src/auto_schedule.cpp

```diff
@@ -231,6 +231,7 @@ void AutoSchedule::init() {
                     m_worker_requests["CPU_HELP"].clear();
                     m_compile_context[CPU].m_compiled_model._ptr.reset();
                     m_compile_context[CPU].m_compiled_model._so.reset();
+                    m_compile_context[CPU].m_is_enabled = false;
```

Comment:
let's check the ptr in the get_property before use.
also add a unit test for this.

Reply:
Updated